### DISCUSSION TITLE
[Question][Hw6] Problem of module String in campaign.ml

### DISCUSSION THREAD

**Discussion by ahnjonggun** [prosyslab-classroom/cs348-information-security#724] [Q&A]
Name: Jaewoo Ahn

When I am using String module in src/campaign.ml like String.length or String.filter, it outputs an error as 
> The module String is an alias for module Stdlib__String, which is missing(ocamllsp)

module String doesn't have any problems in skeleton codes like fuzzer.ml and coverage.ml.

`student@cs348-79:~/cs348-2023-spring-hw6-s20200366/test$ opam switch`
`#  switch                      compiler                                              description`
`   default                     ocaml.4.08.1                                          default`
`→  prosyslab-classroom-4.13.1  ocaml-option-flambda.1,ocaml-variants.4.13.1+options  prosyslab-classroom-4.13.1`
`student@cs348-79:~/cs348-2023-spring-hw6-s20200366$ ls ~/.opam/prosyslab-classroom-4.13.1/lib/ocaml/stdlib__String*`
`/home/student/.opam/prosyslab-classroom-4.13.1/lib/ocaml/stdlib__String.cmi`
`/home/student/.opam/prosyslab-classroom-4.13.1/lib/ocaml/stdlib__String.cmt`
`/home/student/.opam/prosyslab-classroom-4.13.1/lib/ocaml/stdlib__String.cmti`
`/home/student/.opam/prosyslab-classroom-4.13.1/lib/ocaml/stdlib__String.cmx`
`/home/student/.opam/prosyslab-classroom-4.13.1/lib/ocaml/stdlib__StringLabels.cmi`
`/home/student/.opam/prosyslab-classroom-4.13.1/lib/ocaml/stdlib__StringLabels.cmt`
`/home/student/.opam/prosyslab-classroom-4.13.1/lib/ocaml/stdlib__StringLabels.cmti`
`/home/student/.opam/prosyslab-classroom-4.13.1/lib/ocaml/stdlib__StringLabels.cmx`
`

I did like above after looking at https://discuss.ocaml.org/t/the-module-bytes-is-an-alias-for-module-stdlib-bytes-which-is-missing-error-in-ocaml-4-14/11821 
but all my files seem to be in place.
What should I do to solve this problem?
I am using the ssh remote server via vscode.

Thank you

**Comment by yeonhee-ryou** [comment]
If you are working on the KCLOUD VM, you don't need to change the opam switch. Check your source code and the standard `String` module document again.
